packages/common/infra/src/livedata/ops.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  catchError,
+  EMPTY,
+  mergeMap,
+  Observable,
+  type OperatorFunction,
+  pipe,
+} from 'rxjs';
+
+import type { LiveData } from './livedata';
+
+export function mapInto<T>(l$: LiveData<T>) {
+  return pipe(
+    mergeMap((value: T) => {
+      l$.next(value);
+      return EMPTY;
+    })
+  );
+}
+
+export function catchErrorInto(l$: LiveData<any>) {
+  return pipe(
+    catchError((error: any) => {
+      l$.next(error);
+      return EMPTY;
+    })
+  );
+}
+
+export function startOn<T>(cb: () => void): OperatorFunction<T, T> {
```

Comment:
it seems to me the name should be `onStart`, which means to add a callback to the observed observable. Or perhaps something like `tapStart` or even `tapNext`